### PR TITLE
Upgrade decidim 0.16.1

### PR DESCRIPTION
**Pull request by aitorlb**
- Upgrade `decidim 0.16.1`
- Add `rake "decidim:open_data:export"` to `config/schedule.rb`

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,11 +2,10 @@ source "https://rubygems.org"
 
 ruby RUBY_VERSION
 
-DECIDIM_VERSION= '0.16.0'
+DECIDIM_VERSION= '0.16.1'
 
 gem 'decidim', DECIDIM_VERSION
-# gem 'decidim-file_authorization_handler', path: '../decidim-file_authorization_handler.codit'
-gem 'decidim-file_authorization_handler', git: "https://github.com/CodiTramuntana/decidim-file_authorization_handler.git", :branch => 'fix/ensure_all_data_properly_encoded'
+gem 'decidim-file_authorization_handler', git: "https://github.com/MarsBased/decidim-file_authorization_handler.git"
```
Comment (isaacmg410) [inline Gemfile:8]: This are pointing to branch master, is that correct?

**Review by isaacmg410 — COMMENTED** [review]
Good Job. Check my comments please.

File: Gemfile (hunk 1 above)
Comment (aitorlb) [reply Gemfile:8]: Yes

**Review by isaacmg410 — APPROVED** [review]
If there is no other change to do, good work.

Check (and added if is missing) if `config/application.rb` has this line `config.time_zone = 'Madrid'`

**Review by tramuntanal — APPROVED** [review]
Good job @aitorlb !
@isaacmg410 as @aitorlb already answered your question I'm approving so we can move forward.

**Comment by tramuntanal** [comment]
Should we merge @aitorlb ?

**Pull request merged**